R/cancensus.R

```diff
@@ -325,7 +325,21 @@ cancensus.list_vectors <- function(dataset) {
   result %>%
     dplyr::mutate(type = factor(type),
                   units = factor(units)) %>%
-    dplyr::select(vector, type, label, units, parent_vector = parent)
+    dplyr::select(vector, type, label, units, parent_vector = parent) %>%
+    dplyr::mutate(
+      units = recode(.$units, 
+                         `1` = "number",
+                         `2` = "percentage ratio (0.0-1.0)",
+                         `3` = "currency",
+                         `4` = "ratio",
+                         `5` = "percentage (0-100)")) %>%
+    dplyr::mutate(
+      add = recode(.$add,
+                         `1` = "Additive",
+                         `2` = "Averages",
```

Comment:
This is implemented now. The add column returns additive, non additive, average of (var), and median of (var) as possible outcomes now.